wercker.yml

```diff
@@ -42,28 +42,37 @@ build:
 
 deploy:
   steps:
+    - add-to-known_hosts:
+        hostname: github.com
+    - add-ssh-key:
+        keyname: GITHUB_KEY
```

Comment:
We now use ssh to communicate from Wercker to Github.  The public key has been added to Stardust's deploy keys.